### PR TITLE
More updates!

### PR DESCRIPTION
**Pull request by ZacSweers**
Dokka bits, some other dependencies (particularly in the gradle plugin)

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -32,10 +32,23 @@ pluginManagement {
     }
   }
   repositories {
-    gradlePluginPortal()
     mavenCentral()
     google()
-    jcenter()
+    gradlePluginPortal()
```
Comment (gabrielittner) [inline settings.gradle:37]: Not sure if this changed, but at least initially this also served jcenter artifacts so you might want `exclusiveContent` here as well.

**Review by gabrielittner — APPROVED** [review]

File: settings.gradle (hunk 1 above)
Comment (ZacSweers) [reply settings.gradle:37]: removed because I don't think we need anything from jcenter here

File: settings.gradle [hunk 2]
```diff
@@ -32,10 +32,23 @@ pluginManagement {
...
+    exclusiveContent {
+      forRepository {
+        maven {
+          name = "JCenter"
+          setUrl("https://jcenter.bintray.com/")
+        }
+      }
```
Comment (gabrielittner) [inline settings.gradle:44]: At least in kts you can do
```
forRepository { 
  jcenter() 
}
```

Comment (ZacSweers) [reply settings.gradle:44]: 2520c8d

**Pull request merged**